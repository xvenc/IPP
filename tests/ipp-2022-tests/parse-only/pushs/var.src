.IPPcode22
PUSHS GF@var