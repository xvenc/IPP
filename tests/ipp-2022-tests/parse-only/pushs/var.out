<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode22">
 <instruction order="1" opcode="PUSHS">
  <arg1 type="var">GF@var</arg1>
 </instruction>
</program>
